modules/security-network-egress.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * security/container_security/security-network.adoc
+
+[id="security-network-egress_{context}"]
+=  Egress traffic
+
+{product-title} provides the ability to control egress traffic using either
+a router or firewall method. For example, you can use IP whitelisting to control
+database access.
+A cluster administrator can assign one or more egress IP addresses to a project
+in an {product-title} SCN network provider.
```

Comment:
Is SCN correct here, or should it be SDN?

Reply:
Great catch. That was a typo. Fixed.